Gruntfile.js

```diff
@@ -141,6 +141,15 @@ module.exports = function(grunt) {
 				libraries: '**'
 			},
 
+			'library_compat_138': {
+				options: {
+					resources: 'test/preload/fixtures/library',
+					dest: 'tmp/preload/library_compat_138',
+					compatVersion: "1.38"
+				},
+				libraries: '**'
+			},
```

Comment:
Maybe add another `library_compat_140` test case to make sure it will create the JS file.